CMakeLists.txt

```diff
@@ -16,23 +17,23 @@ endif()
 # Add Modules
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/extern/cmake-modules/")
 
-# Add a coverage target iff:
-# - gcovr is available
-# - cesium-native is not being used as a git submodule
-# - g++ is the active compiler.
-# This branch must occur before adding other targets, or gcovr
-# won't be able to inject the necessary coverage information.
-get_directory_property(_is_being_used_as_submodule PARENT_DIRECTORY)
-find_program(_gcovr "gcovr")
-if(CMAKE_COMPILER_IS_GNUCXX AND _gcovr AND NOT _is_being_used_as_submodule)
-    include(CodeCoverage)
-    append_coverage_compiler_flags()
-    setup_target_for_coverage_gcovr_html(
-        NAME cesium-native-tests-coverage
-        EXECUTABLE ctest -j ${PROCESSOR_COUNT}
-        EXCLUDE "${PROJECT_SOURCE_DIR}/extern/*" "${PROJECT_BINARY_DIR}"
-        DEPENDENCIES cesium-native-tests
-    )
+if (CESIUM_COVERAGE_ENABLED)
+    # Add a coverage target iff:
+    # - gcovr is available
+    # - g++ is the active compiler.
+    # This branch must occur before adding other targets, or gcovr
+    # won't be able to inject the necessary coverage information.
+    find_program(_gcovr "gcovr")
+    if(CMAKE_COMPILER_IS_GNUCXX AND _gcovr)
```

Comment:
I think AppleClang work with code coverage too. Maybe we shouldn't check any condition at all here and let the error from `CodeCoverage.cmake` print out?

Reply:
Mac worked out of the box after removing this if statement